darts/models/torch_forecasting_model.py

```diff
@@ -537,8 +503,67 @@ def _save_model(self,
                 for chkpt in checklist[:-1]:
                     os.remove(chkpt)
 
-    def create_dataset(self, series):
-        return _TimeSeriesSequentialDataset(series, self.input_length, self.output_length, self.target_indices)
+    def _prepare_validation_data(self, val_series):
+        if val_series is not None:
```

Comment:
should we add docstring to private method as well ?

Reply:
I asked @TheMP about this as well. I think in general it's not necessary for private methods, especially in this case where the name is quite descriptive.